main.tf

```diff
@@ -3,6 +3,12 @@ locals {
     var.tags,
     {
       "kubernetes.io/cluster/${var.cluster_name}" = "owned"
+    },
+    {
```

Comment:
@omerfsen can you provide references to documentation where you got these annotations from?
thanks

Reply:
https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html I have even setup a test cluster with your terraform. Even if you install cluster-autoscaler without these tags autoscaling does not work

Reply:
https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html